apps/server/src/routes.ts

```diff
@@ -0,0 +1,178 @@
+import type { Route } from './types'
+import {
+  UserApi,
+  DegreeApi,
+  GraphApi,
+  ModuleApi,
+  ModuleCondensedApi,
+} from './api'
+import { body, param, query } from 'express-validator'
+import { validModuleCode } from '@modtree/utils'
+
+export const routes: Route[] = [
+  /**
+   * User API
+   */
+  {
+    method: 'get',
```

Comment:
should be `POST`

Reply:
it's actually `GET`
https://auth0.com/docs/api/management/v2#!/Users/get_users

Reply:
I guess if we aren't using the body then it can be a GET